docker-compose.dev-refresh.yml

```diff
@@ -12,7 +12,7 @@ services:
     ports:
       - "3335:3333"
       - "6061:6061"
-    command: "go run main.go serve grant"
+    command: "cd main && go run main.go serve grant"
```

Comment:
Missing `no-new-privileges:true` in `security_opt`, and `read_only: true`

Reply:
addressed here: https://github.com/brave-intl/bat-go/pull/1593/commits/bb4d7b2dcf9a48fd997fc662a2eb90c2b46d9ae7